taipy/core/_scheduler/_abstract_scheduler.py

```diff
@@ -19,22 +19,31 @@
 class _AbstractScheduler:
     """Creates, Enqueues and schedules jobs as instances of `Job^` class."""
 
+    @classmethod
+    def initialize_scheduler(cls):
```

Comment:
```suggestion
    def initialize(cls):
```
We are in the _AbstractScheduler_, no need to precise _scheduler_ again :)

Reply:
oh yes you're absolutely right!